pages/cockpit/user-guide/manage-environments.adoc

```diff
@@ -0,0 +1,79 @@
+= Manage environments
+:page-sidebar: cockpit_sidebar
+:page-permalink: cockpit/1.x/cockpit_userguide_manage_environments.html
+:page-folder: cockpit/userguide
+:page-description: Gravitee.io Cockpit - Manage environment
+:page-keywords: Gravitee.io, API Platform, API Management, Cockpit, documentation, manual, guide
+
+== Overview
+
+Environments in Cockpit correspond to APIM and AM environments. Environment members and owners described in the procedures below can be users which are already defined with the appropriate role in APIM and AM.
+
+// can't see how the concept of environment is defined in AM
+
+== What can you do with an environment?
+
+You can:
+
+* View details of the environment in the dashboard
+* Update the environment settings
+* Add members
+* Invite members
+* Transfer ownership of the environment
+* Link an installation to the environment
+
+You can view a list of environments in the current organization by clicking the *Environments* menu on the left.
+
+You can update an environment by clicking the edit icon image:icons\cockpit-edit-icon.png[role="icon"] next to the environment in the list, or directly from an environment in the dashboard by clicking the settings icon image:icons\cockpit-settings-icon.png[role="icon"] in the interactive map.
+
+== Update the environment settings
+
+. Click *General*.
+. Update the environment *Name*.
+. Update the environment *Description*.
+. Click *SAVE*.
+. Update the *Human Readable ID* and click *CHANGE HRID*.
+
```

Comment:
There is some new stuff in environment settings: you can now link an environment with an installation. And when installation is linked, you have a button to login to APIM / AM
![screenshot-localhost_4004-2021 02 09-14_01_50](https://user-images.githubusercontent.com/1655950/107367273-67ffca00-6adf-11eb-83ea-17686c36bf08.png)

When you login to an installation from the environments settings, it will open AM / APIM console, scoped in this particular environment.